src/details/ArborX_DetailsDistributor.hpp

```diff
@@ -103,11 +103,16 @@ determineBufferLayout(ExecutionSpace const &space, InputView batched_ranks,
   auto restricted_unique_ranks = InputView(
       compact_ranks, std::make_pair(0, static_cast<int>(n_unique_ranks)));
 
-  auto const unique_ranks_host = Kokkos::create_mirror_view_and_copy(
```

Comment:
Is there a version of `create_mirror_view_and_copy` that does that without intialization? It seems a bit weird to split this function into `create_mirror_view` and `deep_copy` just so that it does not initialize.